packages/migration/src/tests/mods/7.0.0/buttonPropsChange.test.ts

```diff
@@ -0,0 +1,25 @@
+import textMigration from '../../../mods/7.0.0/buttonTextPropChange';
+import iconPropsMigration from '../../../mods/7.0.0/buttonIconPropsPropChange';
+import buttonMenuPropsMigration from '../../../mods/7.0.0/buttonMenuPropsPropChange';
+import buttonToMenuButtonMigration from '../../../mods/7.0.0/buttonToMenuButtonNodeChange';
+import buttonToSplitButtonMigration from '../../../mods/7.0.0/buttonToSplitButtonNodeChange';
+import splitButtonSplitPropRemovalMigration from '../../../mods/7.0.0/splitButtonSplitPropRemovalChange';
+import { runMigration } from '../utils';
+
+describe('buttonPropsChange', () => {
+  it('transforms deprecated props to the corresponding non-deprecated ones', async () => {
+    const result = await runMigration(
+      [
+        textMigration,
+        iconPropsMigration,
+        buttonMenuPropsMigration,
+        buttonToSplitButtonMigration,
+        buttonToMenuButtonMigration,
+        splitButtonSplitPropRemovalMigration
+      ],
+      'button.tsx'
```

Comment:
@jdhuntington Here is the change I made for it to work on Windows. Based on the other examples, this would be `path.join('7.0.0', 'button.tsx')` instead. Changing the other examples also makes them work on Windows.